aria-practices.html

```diff
@@ -3734,6 +3734,390 @@ <h5>Examples</h5>
+          To inform users of assistive technology that this has happened, a polite live region can be used.
+          When the user is done typing, and the search is complete, the user is informed of how many results were found,
+          without having to move focus away from the search field.
+        </p>
+
+        <pre><code>&lt;form role="search" aria-labelledby="search">
+ &lt;h2 id="search">Search&lt;/h2>
+ &lt;label>Search query: &lt;input type="search" name="q" oninput="updateSearch(event)">&lt;/label>
+ &lt;div id="search-result-status" role="region" aria-live="polite">&lt;/div>
+&lt;/form>
+&lt;script>
+async function updateSearch(event) {
+  const statusElement = document.getElementById(&#39;search-result-status&#39;);
+  const results = await getSearchResults(event.target.value);
+  statusElement.textContent = `${results.length} result(s) found.`;
+  showResults(results);
+}
+&lt;/script>
+</code></pre>
+      </section>
+
+      <section id="indicate_what_content_to_announce_with_aria-atomic">
+        <h4>Indicate What Content to Announce with <code>aria-atomic</code></h4>
+
+        <p>
+          The <code>aria-atomic</code> attribute takes the values "true" and "false". The attribute can also be omitted.
+        </p>
+
+        <p>
+          When this attribute is set to <code>true</code>, assistive technologies will render the element as a whole, and not just parts that have been changed.
+        </p>
+
+        <p>
+          When this attribute is set to <code>false</code>, assistive technologies will only render the changes (as per <code>aria-relevant</code>) to the user.
+        </p>
+
+        <p>
+          When this attribute is omitted, the user agent will use the closest ancestor that has <code>aria-atomic</code> set to <code>true</code> or <code>false</code>,
+          or if there is no such ancestor, <code>false</code>.
+        </p>
+
+        <p>
+          For example, consider a clock that can be set to notify the user of the current time at a regular interval.
+          Without <code>aria-atomic</code>, the assistive technology might only notify the changed components, rather than the full time.
+        </p>
+
+        <pre><code>&lt;div id="clock" role="region" aria-live="polite" aria-atomic="true">
+ The time is
+ &lt;span>16&lt;/span>:&lt;span>34&lt;/span>:&lt;span>05&lt;/span>
+&lt;/div>
+</code></pre>
+      </section>
+
+      <section id="indicate_which_content_changes_are_relevant_with_aria-relevant">
+        <h4>Indicate Which Content Changes are Relevant with <code>aria-relevant</code></h4>
+
+        <p>
+          The <code>aria-relevant</code> attribute can be used to inform assistive technologies about which kinds of changes are relevant to inform users about.
+          It takes a list of keywords, with the following meanings:
+        </p>
+
+        <ul>
+          <li><code>additions</code>: Element nodes are added to the accessibility tree within the live region.</li>
```

Comment:
```suggestion
          <li><code>additions</code>: When element nodes are added to the accessibility tree within the live region.</li>
```

Consider adding "When" to these (short for "Relevant when...").